__mocks__/handlers/task-details.handler.ts

```diff
@@ -32,7 +32,7 @@ const taskDetailsHandler = [
                     },
                     github: {
                         issue: {
-                            html_url:'https://www.sampleGithubUrl.com'
+                            html_url:'https://github.com/sampleOrg/sampleRepo/issues/000'
```

Comment:
why the url is camelcase?

Reply:
@gauravsinhaweb No specific reason; casing doesn't matter for HTML URLs; it's just for better readability.

Reply:
> @gauravsinhaweb No specific reason; casing doesn't matter for HTML URLs; it's just for better readability.
Have a look at this documentation and share me you thoughts on this
https://api.gov.au/sections/naming-conventions.html#:~:text=only%20hyphens%20'%2D'%20can%20be,part%20of%20the%20base%20URI

Reply:
have a look at this documentation and share me your thoughts on this
https://api.gov.au/sections/naming-conventions.html#:~:text=only%20hyphens%20'%2D'%20can%20be,part%20of%20the%20base%20URI

Reply:
@gauravsinhaweb Thanks for properly highlighting and sharing the doc. I have updated the URL to follow the convention.